yb-voyager/cmd/exportData.go

```diff
@@ -137,7 +137,24 @@ func exportDataCommandFn(cmd *cobra.Command, args []string) {
 		utils.PrintAndLogf("Note: Beta feature to accelerate data export is enabled by setting BETA_FAST_DATA_EXPORT environment variable")
 	}
 	if changeStreamingIsEnabled(exportType) {
-		utils.PrintAndLog(color.YellowString(`Note: Live migration is a TECH PREVIEW feature.`))
+		switch source.DBType {
+		case ORACLE:
+			utils.PrintAndLogfWarning("Note: Live migration is a TECH PREVIEW feature.")
+		default:
+			if exporterRole == SOURCE_DB_EXPORTER_ROLE {
+				utils.PrintAndLogfWarning("\nImportant: The following limitations apply to live migration:\n")
+				utils.PrintAndLogfInfo("  1. Schema modifications on the source and target databases are not supported during live migration.\n")
+				utils.PrintAndLogfInfo("  2. Modifying key columns (Primary Key or Unique Key) between source and target databases is not supported.\n")
+				utils.PrintAndLogfInfo("  3. Partition management operations (adding or deleting partitions) on partitioned tables are not supported.\n")
```

Comment:
Shall we  include this in the first point as an additional detail? 

Schema modifications (for example, adding/droping columns, creating/deleting tables, creating/deleting partitions, etc) on the source and target database are not supported during live migration.